third_party/terraform/tests/resource_compute_subnetwork_test.go

```diff
@@ -402,6 +402,7 @@ resource "google_compute_subnetwork" "network-ref-by-url" {
   ip_cidr_range = "10.0.0.0/16"
   region        = "us-central1"
   network       = google_compute_network.custom-test.self_link
+//  private_ipv6_google_access = "ENABLE_OUTBOUND_VM_ACCESS_TO_GOOGLE"
```

Comment:
Is this commented out intentionally?

Reply:
yes for now. enabling it breaks the test.

Reply:
Is this still a work in progress, then?  Would you mind setting it to one, if it is?
https://github.blog/2019-02-14-introducing-draft-pull-requests/

Reply:
Can I submit it without the field? I can set it if the project you are running the tests in have sufficient quota.

Reply:
Except it is going to break anyone else who hasn't had their projects enabled for this.

Reply:
Google API sets it to a default value but it makes no sense in setting that value in the test.

Reply:
It can't be submitted with commented-out code, but I'm okay with submitting it without testing this particular case if you can demonstrate that it works.  :)

Reply:
done, tests were passing with the field commented out.